cross/icecast/Makefile

```diff
@@ -0,0 +1,19 @@
+PKG_NAME = icecast
+PKG_VERS = 2.4.1
+PKG_EXT = tar.gz
+PKG_DIST_NAME = $(PKG_NAME)-$(PKG_VERS).$(PKG_EXT)
+PKG_DIST_SITE = http://downloads.xiph.org/releases/icecast
+PKG_DIR = $(PKG_NAME)-$(PKG_VERS)
+
+DEPENDS = cross/libxml2 cross/openssl cross/curl cross/libvorbis cross/libxslt cross/speex cross/libtheora
+
+HOMEPAGE = http://www.icecast.org/
+COMMENT  = Icecast is a streaming media server which currently supports Ogg (Vorbis and Theora), Opus, WebM and MP3 audio streams.
+LICENSE  = GPLv2
+
+GNU_CONFIGURE = 1
+
+#CONFIGURE_ARGS = --with-curl="$(INSTALL_DIR)$(INSTALL_PREFIX)" --with-ogg="$(INSTALL_DIR)$(INSTALL_PREFIX)" --with-vorbis="$(INSTALL_DIR)$(INSTALL_PREFIX)" --with-theora="$(INSTALL_DIR)$(INSTALL_PREFIX)" --with-speex="$(INSTALL_DIR)$(INSTALL_PREFIX)" --with-openssl="$(INSTALL_DIR)$(INSTALL_PREFIX)"
```

Comment:
if i recall that correctly i didn't had to use this (maybe because i have a native xslt-config and that was used) but i'll try again to be sure :)